### PR TITLE
{bio}[gompi/2020b] HISAT2 v2.2.1, NGS v2.10.9, ncbi-vdb v2.10.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/05e235a65ef426a13320cdc4413c4967 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/6f7a5df2f0cda20fb885caa051ff0556 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/7496da9d9a50a88982fec657ed1b3df0 for a full test report.

File: easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
```
Comment (verdurin) [inline easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCCcore-10.2.0.eb:1]: Missing an author attribution from previous versions here.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node304.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/eef7589724872919e9d90708055d4036 for a full test report.

File: easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCCcore-10.2.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCCcore-10.2.0.eb:1]: This easyconfig is based on `NGS-2.10.4-GCCcore-8.3.0-Java-11.eb` (which I authored) so it's ok as it is :)

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**